### PR TITLE
Improve Testerina object mocking tests

### PR DESCRIPTION
**Pull request by Ibaqu**
## Purpose
Improve Testerina object mocking tests

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: devtools-integration-tests/testerina-integration-tests/src/testerina-object-mocking-test/client.bal [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import ballerina/http;
+
+// Global HTTP Client
+http:Client orderManagementClient = new("http://localhost:9117/ordermgt");
+
+// Gloabal response payload for each function
+json responsePayload = {};
+
+// Look search for the order in the system
```
Comment (Dilhasha) [inline devtools-integration-tests/testerina-integration-tests/src/testerina-object-mocking-test/client.bal:9]: @Ibaqu Shall we get rid of comments from places where it's self explanatory?

File: devtools-integration-tests/testerina-integration-tests/src/testerina-object-mocking-test/client.bal [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import ballerina/http;
+
+// Global HTTP Client
```
Comment (Dilhasha) [inline devtools-integration-tests/testerina-integration-tests/src/testerina-object-mocking-test/client.bal:3]: @Ibaqu Shall we get rid of comments from places where it's self explanatory?

Comment (Ibaqu) [reply devtools-integration-tests/testerina-integration-tests/src/testerina-object-mocking-test/client.bal]: removed comments

File: devtools-integration-tests/testerina-integration-tests/src/testerina-object-mocking-test/service.bal [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    }
+    resource function find(http:Caller caller, http:Request req, string orderId) {
+        // Find the requested order from the map and retrieve it in JSON format.
+        json? payload = ordersMap[orderId];
+        http:Response response = new;
+        if (payload == null) {
+            payload = { "Error" : "Order : " + orderId + " cannot be found." };
+        }
+
+        // Set the JSON payload in the outgoing response message.
```
Comment (Dilhasha) [inline devtools-integration-tests/testerina-integration-tests/src/testerina-object-mocking-test/service.bal:30]: @Ibaqu Shall we get rid of comments from code where it's self explanatory

File: devtools-integration-tests/testerina-integration-tests/src/testerina-object-mocking-test/tests/client_test.bal [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    };
+
+    json actual = addOrder(orderPayload);
+    json expected = {"status":"Order Created.","orderId":"100500"};
+
+    test:assertEquals(actual, expected, "Add order test failed");
+
+}
+
+@test:Config { enable : false}
```
Comment (Dilhasha) [inline devtools-integration-tests/testerina-integration-tests/src/testerina-object-mocking-test/tests/client_test.bal:40]: Why are these tests disabled?

File: devtools-integration-tests/testerina-integration-tests/src/testerina-object-mocking-test/tests/client_test.bal [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+    }
+};
+
+json getResponse2 = { "Error" : "Order : 200 cannot be found." };
+
+@test:Config { }
+function test_findOrder() {
+
+    // Create a Specific response for get
+    http:Response mockGetResponse = new;
```
Comment (Dilhasha) [inline devtools-integration-tests/testerina-integration-tests/src/testerina-object-mocking-test/tests/client_test.bal:73]: Shall we move the mock  response creation to a separate function?

**Comment by Ibaqu** [comment]
Closing PR and redoing it for the project-api-update branch

**Pull request closed without merging**